engine/backends/proverif/proverif_backend.ml

```diff
@@ -285,6 +285,24 @@ module Print = struct
                 | None -> super#pat' ctx pat)
             | _ -> super#pat' ctx pat
 
+      method! expr_app f args _generic_args =
+        let args =
+          separate_map
+            (comma ^^ break 1)
+            (print#expr_at Expr_App_arg >> group)
+            args
+        in
+        let f =
+          match f with
+          | { e = GlobalVar name; _ } -> (
+              match name with
+              | `Projector (`Concrete i) | `Concrete i ->
+                  print#concrete_ident' ~under_current_ns:false i |> group
```

Comment:
Do you need this `print#concrete_ident' ~under_current_ns:false` (insead of just using `print#concrete_ident ...`), since you overrided `concrete_ident`?

Reply:
You're right, that's not necessary.